packages/trip-details/src/fare-table.tsx

```diff
@@ -89,7 +89,7 @@ const FareTypeTable = ({
       ...col,
       total: getItineraryCost(legs, col.mediumId, col.riderCategoryId)
     }))
-    .filter(col => col.total);
+    .filter(col => col.total !== undefined);
```

Comment:
Should there be a story to illustrate this new case?

Reply:
I thought about it, but I think it's too small of a corner case and adding another story for it would require another itinerary, so I don't think it's worth it.